src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -430,6 +466,22 @@ public void arcadeDrive(double speed, double rot) {
     setSpeeds(wheelSpeeds);
   }
 
+  /**
+   * Controls the left and right sides of the drive directly with voltages.
+   *
+   * @param leftVolts the commanded left output
+   * @param rightVolts the commanded right output
+   */
+  public void tankDriveVolts(double leftVolts, double rightVolts) {
+    SmartDashboard.putNumber("Volts Left", leftVolts);
+    SmartDashboard.putNumber("Volts Right", rightVolts);
```

Comment:
Are these needed? If they are meant to be permanent, please change them to shuffleboard

Reply:
Just for debug.  Please provide the Shuffleboard equivalent.

Reply:
For debug might be nice to put it in network tables using this style: 
- https://github.com/BHSRobotix/ChargedUp2023/blob/17bcb4606dca5febe827852418c6f4625e5056c1/src/main/java/bhs/devilbotz/subsystems/DriveTrain.java#L232
- https://github.com/BHSRobotix/ChargedUp2023/blob/17bcb4606dca5febe827852418c6f4625e5056c1/src/main/java/bhs/devilbotz/subsystems/DriveTrain.java#L274

This groups all entries in single table, which makes it easy to add to shuffleboard and view in a single tab.  And much easier than adding a bunch of code add/place all the boxes/widgets.  Drivers are not going to care to view any of this debug output so we don't have to worry about arranging it in some pretty way.